src/App.js

```diff
@@ -1,11 +1,96 @@
-import './App.css';
+// modules
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+// components
+import PageHeading from "./components/PageHeading.js";
+import DisplayPhotos from './components/DisplayPhotos.js';
+import Button from './components/Button.js';
+import PageFooter from './components/PageFooter.js';
+// // styling
+import './styles/sass/App.scss';
+
 
 function App() {
+
+  // useState objects
+  const [message, setMessage] = useState("");
+  const [author, setAuthor] = useState("");
+  const [allPhotos, setAllPhotos] = useState([]);
+  const [randomizer, setRandomizer] = useState(false);
+
+  // a function to trigger a button submission
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setRandomizer(!randomizer);
+  }
+  
+  // a function to get a min an max value between two numbers
+  const getRandomInt = ((min, max) => {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min) + min); 
```

Comment:
great thinking to create this function globally in your app component so you can use it anywhere you need!